app/views/provider_interface/content/rendered_markdown_template.html.erb

```diff
@@ -1,17 +1,17 @@
 <%= content_for :title, t("page_titles.#{@page_name}") %>
 
-<% if @with_breadcrumbs %>
+<% if @breadcrumb_title %>
```

Comment:
should we be checking for title and path?

Reply:
I was thinking that they could be bundled as an array or something, so that it's just one param - but we want both or neither, so I figured checking for one was sufficient. I'll add an additional check for the path as well, as it's not suitable to render anything without it anyway.